lantern-ui/app/js/services.js

```diff
@@ -90,6 +90,9 @@ angular.module('app.services', [])
           window.location = settings.redirectTo;
         }
       },
+      'bandwidth': function(bandwidth) {
+        console.log('Got bandwidth data: ', bandwidth);
```

Comment:
@myleshorton Bandwidth data is plumbed through to web browser.  Can you take it from here and bind into the UI as appropriate?